scripts/customer/openAcc.py

```diff
@@ -0,0 +1,29 @@
+# Bailee Segars
+from datetime import date
+from decimal import *
```

Comment:
```suggestion
from decimal import Decimal
```
import only what we need from module